docs/contents/getting_started/index.md

```diff
@@ -68,7 +68,6 @@ $ cd pdf.js
 │   └── pdf.worker.js                      - core layer
 └── web/
     ├── cmaps/                             - character maps(required by core)
-    ├── compatibility.js                   - polyfills for missing features
```

Comment:
This was removed in PR #8819; please rebase your patch to the *current* `master`.

Reply:
Done, sorry for the wait

Reply:
We don't allow merge commits in PRs, please squash this into *one commit*; see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits for additional information.

Reply:
You taught me something today! ;) Done. Thanks!